src/index.html

```diff
@@ -7,6 +7,641 @@
+            </li>
+            <li class="header__item">
+              <a href="#smartphones" class="header__link">Smartphones</a>
+            </li>
+            <li class="header__item">
+              <a
+              href="#laptops&computers"
+              class="header__link"
+              data-qa="nav-hover"
+              >
+              Laptops & Computers
+              </a>
+            </li>
+            <li class="header__item">
+              <a href="#gadgets" class="header__link">Gadgets</a>
+            </li>
+            <li class="header__item">
+              <a href="#tablets" class="header__link">Tablets</a>
+            </li>
+            <li class="header__item">
+              <a href="#photo" class="header__link">Photo</a>
+            </li>
+            <li class="header__item">
+              <a href="#video" class="header__link">Video</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main>
+      <section class="catalog">
+        <div class="catalog__container">
+          <div class="catalog__wrapper">
+            <article data-qa="card" class="catalog__item product-card">
+              <div class="product-card__image">
+                <img
+                  class="product-card__img"
+                  src="./images/imac.jpeg"
+                  alt="imac">
+              </div>
+
+              <h2 class="product-card__title">
+                  APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+              </h2>
+
+              <p class="product-card__code">
+                  Product code: 195434
+              </p>
+
+              <div class="product-card__stars">
+                <div class="product-card__stars-box stars stars--4">
+                  <div class="stars__star"></div>
+
+                  <div class="stars__star"></div>
+
+                  <div class="stars__star"></div>
+
+                  <div class="stars__star"></div>
+
+                  <div class="stars__star"></div>
+                </div>
+
+                <p class="product-card__review">
+                    Reviews: 5
+                </p>
+              </div>
+              <div class="product-card__price">
+
+                <p class="product-card__price-text">
+                    Price:
+                </p>
+
+                <div class="product-card__price-value">
+                    $2,199
+                </div>
+              </div>
+
+              <div class="product-card__link">
+                <a
+                href="#"
+                class="product-card__url"
+                data-qa="card-hover"
+                >
+                Buy
+                </a>
+              </div>
+            </article>
+            <article class="catalog__item product-card">
+              <div class="product-card__image">
+                <img
+                class="product-card__img"
+                src="./images/imac.jpeg"
+                alt="imac">
```

Comment:
You need to check and fix it everywhere
```suggestion
                <img
                  class="product-card__img"
                  src="./images/imac.jpeg"
                  alt="imac"
                >
```
```suggestion
                <img
                class="product-card__img"
                src="./images/imac.jpeg"
                alt="imac">
```